R/FilterFindCorrelation.R

```diff
@@ -0,0 +1,80 @@
+#' @title Correlation Filter
+#'
+#' @usage NULL
+#' @name mlr_filters_correlation
+#' @format [R6::R6Class] inheriting from [Filter].
+#' @include Filter.R
+#'
+#' @section Construction:
+#' ```
+#' FilterFindCorrelation$new()
+#' mlr_filters$get("findcorrelation")
+#' flt("findcorrelation")
+#' ```
+#'
+#' @description
+#' Simple filter emulating `caret::findCorrelation(exact = FALSE_)`.
+#' 
+#' This gives each feature a score that is the *negative* of the cutoff value
+#' for which it is excluded when using `caret::findCorrelation()`. The negative
+#' is used because `caret::findCorrelation()` excludes everything *above* a cutoff,
+#' while filters exclude everything below a cutoff. Therefore, `caret::findCorrelation(cutoff = 0.9)`
+#' lists the same features that are excluded with `FilterFindCorrelation` at score -0.9.
+#'
+#' @family Filter
+#' @template seealso_filter
+#' @export
+#' @examples
+#' ## Pearson (default)
+#' task = mlr3::tsk("mtcars")
+#' filter = flt("findcorrelation")
+#' filter$calculate(task)
+#' as.data.table(filter)
+#'
+#' ## Spearman
+#' filter = FilterFindCorrelation$new()
+#' filter$param_set$values = list("method" = "spearman")
+#' filter$calculate(task)
+#' as.data.table(filter)
+FilterFindCorrelation = R6Class("FilterFindCorrelation", inherit = Filter,
+  public = list(
+    initialize = function() {
+      super$initialize(
+        id = "correlation",
```

Comment:
We already have a filter with `id = "correlation"`.

Can we find a more matching name that deviates clearly from the existing one?

Reply:
This is obviously a typo, it should be `"findcorrelation"`, because it imitates the `caret::findCorrelation` function. What other name would you suggest / what naming scheme do you use in mlr3filters?